frontend/src/components/forms/CreateLog.tsx

```diff
@@ -353,14 +347,17 @@ const CreateLog = () => {
                       placeholder="Select Resident"
                       onChange={handleResidentChange}
                       styles={{
```

Comment:
Update the styles to only use selectStyle
```suggestion
                      styles={selectStyle}
```